week_1/week1_nanda_kishor_vinod.html

```diff
@@ -0,0 +1,82 @@
+
+<style>
+    .title{
+        font-weight:bold;
+        font-size:30px;
+    }
+</style>
+<style>
+    .namebox{
+        margin-right:100px;
+    }
+</style>
+<style>
+    .emailbox{
+        margin-right:43px;
+    }
+</style>
+<style>
+    .sbox{
+        height:125px;
+        width:300px;
+        font-size:30px;
+        font-weight:50;
+        
+    }
+</style>
+<style>
+    .d{
+        margin-right:400px;
+    }
+</style>
+
+
+<h1 class="title">
+    My Guest Book
+</h1>
+<p>
+    Please sign my guest book. Thanks!
+</p>
+<label for="name" class="namebox"><b>What is your name?</b></label>
+<input type="text" placeholder="Your name"><br>
+<br>
+<label for="email" class="emailbox"><b>What is your email address?</b></label>
+<input type="text" placeholder="Your email"><br>
+<br>
+<br>
+<label for="atapply"><div class="d" ><b>Please check all that apply: </b></div></label>
+<input type="checkbox" id="a">I really like your website<br>
+<input type="checkbox" id="b">One of the best sites I've seen<br>
+<input type="checkbox" id="c">I sure wish my site looked as good as yours. <br>
+<input type="checkbox" id="d">I have no taste and I am pretty dense,so your site didn't do much for me<br>
+<br>
+<br>
+<label for="ctb2"><div class="d"><b>Choose the one thing that you love best about my website</b></div></label>
+<input type="radio" id="h">That gorgoeus picture of you 
+<br>
+<input type="radio" id="h">All the beautiful pictures of your cats
+<br>
+<input type="radio" id="h ">The inspiring recap of your suburban childhood
+<br>
+<input type="radio" id="h " >The detailed list of all your Elvis memorabilia's
```

Comment:
IDs are supposed to be unique. You can read more [here](https://www.w3schools.com/tags/att_input_type_radio.asp) about how to implement radio boxes